ros/manifest.yaml

```diff
@@ -274,6 +261,26 @@ release_names:
                             perception:
                                 aliases:
                                     - "$release_name-perception-$os_code_name"
+            debian:
+                os_code_names:
+                    stretch:
+                        <<: *DEFAULT_ROS1
+                        archs:
+                            - amd64
+                            - arm64v8
+                        tag_names:
+                            # ros-core:
```

Comment:
> Current blocker for this PR:
The ROS Lunar / Debian Stretch packages are [not available in the snapshot repository](http://snapshots.ros.org/lunar/final/debian/dists/stretch/) (issue raised [here](https://discourse.ros.org/t/announcing-ros-snapshot-repositories/7705/11)). 

Pinning a reminder to uncomment here when resolved.

Reply:
That's exciting and weird.

Reply:
great! thanks @nuclearsandwich for syncing them.
Updated the PR accordingly and it builds flawlessly :+1: